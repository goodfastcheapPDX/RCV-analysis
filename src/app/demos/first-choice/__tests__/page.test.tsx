import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import FirstChoiceDemoPage from "../page";

describe("FirstChoiceDemoPage", () => {
  it("should match snapshot", () => {
    const { container } = render(<FirstChoiceDemoPage />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
