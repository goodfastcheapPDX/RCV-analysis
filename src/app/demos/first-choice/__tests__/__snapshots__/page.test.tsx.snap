// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FirstChoiceDemoPage > should match snapshot 1`] = `
<div
  class="container mx-auto py-8 px-4"
>
  <div
    class="max-w-6xl mx-auto"
  >
    <div
      class="mb-8"
    >
      <h1
        class="text-3xl font-bold"
      >
        First Choice Breakdown Demo
      </h1>
      <p
        class="text-gray-600 mt-2"
      >
        Loading first-choice vote breakdown visualization...
      </p>
    </div>
    <div
      class="flex items-center justify-center py-12"
    >
      <div
        class="animate-spin rounded-full h-8 w-8 border-b-2 border-blue-600"
      />
    </div>
  </div>
</div>
`;
